Description: lib/pcblib-newlib is in TOPSRCDIR not in TOP_BUILDDIR
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@sabily.org>
Bug: https://bugs.launchpad.net/pcb/+bug/855405
Forwarded: https://bugs.launchpad.net/pcb/+bug/855405/+attachment/2435173/+files/pcbtest_paths.diff
--- a/src/pcbtest.sh.in
+++ b/src/pcbtest.sh.in
@@ -37,7 +37,7 @@
 # Use  --g-fatal-warnings with the gtk HID to cause gtk-WARNING's to
 # abort.
 
-TEST_PATHS="--lib-path @TOP_BUILDDIR@/lib --lib-newlib @TOPSRCDIR@/newlib:@TOP_BUILDDIR@/lib/pcblib-newlib --element-path @TOP_BUILDDIR@/lib --font-path @TOPSRCDIR@/src"
+TEST_PATHS="--lib-path @TOP_BUILDDIR@/lib --lib-newlib @TOPSRCDIR@/newlib:@TOPSRCDIR@/lib/pcblib-newlib --element-path @TOP_BUILDDIR@/lib --font-path @TOPSRCDIR@/src"
 TEST_CMDS="--lib-command-dir @TOP_BUILDDIR@/lib"
 
 # note:  To do command line exporting, pcb requires the "-x <export_hid>" command to appear first.  For example
